src/main/scala/com/mwz/sonar/scala/sensor/Measures.scala

```diff
@@ -21,10 +21,15 @@ package com.mwz.sonar.scala.sensor
 import scala.annotation.tailrec
 import scalariform.lexer.{Token, Tokens}
 
+import scala.util.matching.Regex
+
 /** Scala Sensor Metrics */
 object Measures {
+
+  val NewLineRegex: Regex = "(\r\n)|\r|\n".r
+
   def countClasses(tokens: List[Token]): Int = {
-    tokens.foldLeft(0) {
+    tokens.foldLeft(0) { // scalastyle:ignore LiteralArguments
```

Comment:
+1 for the comments to disable the **LiteralArguments** _rule_, I didn't knew it.
Maybe there are some places in the `scoverage plugin` where we should use this? I remember a discussion about this same rule when you reviewed #34.

Reply:
Yes possibly, although Sonar is not picking any of those in the scoverage plugin, so it looks like you must have fixed them all.

Reply:
Let's merge this PR and see what issues we have left.